test/utils/bytecode.hpp

```diff
@@ -524,9 +524,9 @@ struct create_instruction
     }
 
     template <Opcode k = kind>
-    typename std::enable_if<k == OP_CREATE4, create_instruction&>::type initcode(uint8_t index)
+    typename std::enable_if<k == OP_CREATE4, create_instruction&>::type initcode(bytecode hash)
```

Comment:
Argument shouldn't be `bytecode`, because this is just immediate 32 bytes and we don't need here the flexibility of any bytecode producing this argument as for regular opcodes with stack arguments. So I thihk it should be just `bytes32` or `hash256`, same for `m_initcode_hash` member.